color/src/colorspace.rs

```diff
@@ -1515,4 +1646,30 @@ mod tests {
             ));
         }
     }
+
+    #[test]
+    fn absolute_conversion() {
+        assert!(almost_equal::<AcesCg>(
+            Srgb::convert_absolute::<AcesCg>([0.5, 0.2, 0.4]),
+            // Calculated using colour-science (https://github.com/colour-science/colour) with
```

Comment:
I tried adding `colour-science = "colour-science"` to `.typos.toml`'s `[default.extend-identifiers]` section, but it keeps triggering (locally, at least). I think `typos` doesn't handle hyphens, so it's just triggering on `colour = "color"`.

Reply:
Solved, thanks @DJMcNab!